devops/docker/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM base/archlinux:latest
```

Comment:
Are there any docs about running this from within Docker (including the mounting and so on)? If not can we write up a small paragraph?